matrix_client/client.py

```diff
@@ -667,3 +680,14 @@ def remove_room_alias(self, room_alias):
             return True
         except MatrixRequestError:
             return False
+
+    def get_account_data(self):
```

Comment:
Hey, sorry for the late response, @Zil0. I chose to have another look.

I chose this particular formatting because it matches the JSON that the API returns. But looking at the `api.set_account_data` call, you're right, I can probably just return the `account_data` dict, do yo agree?